internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1230,6 +1419,16 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(ctx context.
 				ContainerPort: int32(servicePort.IntValue()),
 				Protocol:      corev1.ProtocolTCP,
 			},
+			{
+				ContainerPort: int32(serviceHTTPPort.IntValue()),
+				Protocol:      corev1.ProtocolTCP,
+			},
+		},
+		Command: []string{
+			"/assisted-service",
```

Comment:
Perhaps we should convert Dockerfile to use `ENTRYPOINT` so that we would not have to define it here?